configure.ac

```diff
@@ -562,6 +562,7 @@ if test "$enable_cloud" = "yes"; then
         HAVE_libmosquitto_a="yes"
     else
         HAVE_libmosquitto_a="no"
+        AC_DEFINE([ACLK_NO_LIBMOSQ], [1], [Libmosquitto.a was not found during build.])
```

Comment:
Why not just put the HAVE_ flags straight into the config.h?

Reply:
can but don't see any advantage